software/eudaq/include/eudaq/CSVParser.h

```diff
@@ -0,0 +1,178 @@
+#ifndef EUDAQ_CSVPARSER_H
+#define EUDAQ_CSVPARSER_H
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <iterator>
+#include <boost/tokenizer.hpp>
```

Comment:
Is Boost installed on the CERN computers?

Reply:
According to PF, boost is on the CERN computers.